### ISSUE TITLE
Add Mechanism to Refetch in `createMutatorStore` Without a Loading State

### ISSUE THREAD

**Issue by martinmckenna** [nanostores/query#25]
Hi I have a use case like this:

```ts
const queryKey = "things"

export const $things = createFetcherStore(queryKey, {
      fetcher: () =>
        getThings().then((response) => response.data),
    })

export const $createThing = createMutatorStore<Payload>(({ data, invalidate }) => {
  invalidate(queryKey);
  return createThing(payload);
});
```

I would really like a way for `createMutatorStore` to be able to call `invalidate` to trigger a refetch but without setting `$things.loading` back to `true`. I want it to replace the old data with the new data but do it without clearing everything out before.

Either that, or expose a `refetch` callback on the `createFetcherStore` return value, so that we can call it from the react component

This seems pretty standard in libraries like react query

**Comment by martinmckenna** [comment]
ah nvm I guess you can just say `if (!data && loading)` as the condition for first load

**Comment by dkzlv** [comment]
@martinmckenna Yep, you guessed it right! I suppose, it'd be better to update the README to reflect a better approach then `if (loading) return <>loading</>`. I [already use](https://stackblitz.com/edit/react-ts-cend2y?file=App.tsx) a different approach in all my SB demos.
